--- keystonemiddleware/auth_token/__init__.py
+++ keystonemiddleware/auth_token/__init__.py
@@ -940,11 +940,17 @@ class AuthProtocol(BaseAuthProtocol):
         return session_loading.Session().load_from_options(**kwargs)

     def _create_identity_server(self):
+        try:
+            endpoint_override=self._conf.get('endpoint_override')
+        except cfg.NoSuchOptError:
+            endpoint_override = None
+
         adap = adapter.Adapter(
             self._session,
             auth=self._auth,
             service_type='identity',
             interface='admin',
+            endpoint_override=endpoint_override,
             region_name=self._conf.get('region_name'),
             connect_retries=self._conf.get('http_request_max_retries'))
